test/test_torch.py

```diff
@@ -16094,43 +16094,55 @@ def _test(use_out, row_major, incx, incy, lda_tail):
         for use_out, row_major, incx, incy, lda_tail in product((False, True), (False, True), (1, 2), (1, 2), (0, 1)):
             _test(use_out, row_major, incx, incy, lda_tail)
 
-    @slowTest
-    @onlyCPU
-    def test_addmm(self, device):
-        dtypes = {
+    @dtypesIfCUDA(*torch.testing.get_all_complex_dtypes(), *torch.testing.get_all_fp_dtypes(include_bfloat16=False))
+    @dtypes(*torch.testing.get_all_complex_dtypes(), *torch.testing.get_all_fp_dtypes())
+    def test_addmm(self, device, dtype):
+        prec = {
```

Comment:
can you use @precisionOverrides for this?

Reply:
Fixed. And note that the precision for bfloat16 is bumped to 0.5